kobo/settings/base.py

```diff
@@ -580,7 +579,18 @@ def __init__(self, *args, **kwargs):
 if env.str('STRIPE_TEST_SECRET_KEY', None) or env.str('STRIPE_LIVE_SECRET_KEY', None):
     STRIPE_ENABLED = True
 
+
+def dj_stripe_request_callback_method():
+    # This method exists because dj-stripe's documentation doesn't reflect reality.
```

Comment:
Maybe link to the upstream error report here? This hack could be removed if it gets fixed upstream.